pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/generator/PinotTaskGenerator.java

```diff
@@ -66,6 +66,13 @@ default int getNumConcurrentTasksPerInstance() {
     return JobConfig.DEFAULT_NUM_CONCURRENT_TASKS_PER_INSTANCE;
   }
 
+  /**
+   * Returns the maximum number of attempts per task, 1 by default.
+   */
+  default int getMaxAttemptsPerTask() {
+    return 1;
```

Comment:
Would like to make this a constant but not sure where to add it.

Reply:
You may put it under the `MinionConstants`

Reply:
Created under `MinionConstants`.